include/deal.II/cgal/utilities.h

```diff
@@ -103,6 +115,42 @@ namespace CGALWrappers
   cgal_surface_mesh_to_cgal_coarse_triangulation(
     CGAL::Surface_mesh<typename C3t3::Point::Point> &surface_mesh,
     C3t3 &                                           triangulation);
+
+  /**
+   * Given two triangulated surface meshes, the corefinement operation consists
+   * in refining both meshes so that their intersection polylines are a subset
+   * of edges in both refined meshes. The corefinement of two triangulated
+   * surface meshes can naturally be used for computing Boolean operations on
+   * volumes. The last parameter drives the selection of the boolean operation
+   * that one wants to perform, and can be `BooleanOperation::corefine`,
+   * `BooleanOperation::difference12`, `BooleanOperation::difference21`,
+   * `BooleanOperation::intersection`, `BooleanOperation::merge` if one wants to
+   * compute the corefinement only, differences, intersection or union of the
+   * two meshes, respectively. In case of corefinement only, the operation will
+   * be performed directly on the original meshes `surf_1` and `surf_2`. The
+   * shaded region in the following picture shows the result of the corefinement
+   * and intersection between a cube and the green polyhedra
+   *
+   * @image html corefine_and_compute_intersection.png
```

Comment:
Can you trim the image so that there is no axis in the image, and no (or little) white space around the image?

Also, would you mind adding an example image for each of the supported operations, so that it is clear what the other operations do as well?